File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Vision/Models/PatientRecord/Name.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Vision.Models.PatientRecord
{
    public class Name 
    {
        [XmlElement(ElementName="title", Namespace="urn:vision")]
        public string Title { get; set; }

        [XmlElement(ElementName="forename", Namespace="urn:vision")]
        public string Forename { get; set; }
        
        [XmlElement(ElementName="surname", Namespace="urn:vision")]
        public string Surname { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/vision/demographics/Name.kt
package mocking.vision.demographics

import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name="name", namespace = "urn:vision")
data class Name(@XmlElement(namespace = "urn:vision", name = "title")
                var title: String,
                @XmlElement(namespace= "urn:vision",name = "forename")
                var forename: String,
                @XmlElement(namespace= "urn:vision",name = "surname")
                var surname: String)
{
    constructor() : this("", "", "")
}
